index.js

```diff
@@ -87,11 +72,24 @@ class Main {
     newActivePage.classList.remove('hidden');
     newActivePage.classList.add('opened');
   }
+
+  getLocalTime = () => {
+    const time = luxon.DateTime.now().toLocaleString(luxon.DateTime.DATETIME_MED_WITH_SECONDS);
+    return time;
+  }
+
+  getTime = () => {
+    setInterval(() => {
+      timeIndicator.innerHTML = this.getLocalTime();
+    }, 1000);
+  }
 }
 
 const mainClass = new Main();
 mainClass.formSubmit();
 mainClass.loadData();
+mainClass.getTime();
+
 navigationLinks.forEach((el) => {
   el.addEventListener('click', () => {
     document.querySelector('.active').classList.remove('active');
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirements, please make sure your app is working as expected, currently, User can't `add a book` (and impossible to test the `remove` action as there is no added book), the` current date and time is not displaying`, and the `contact page doesn't work`. Make sure you fix those issues to match the expected result.
I recommend you first test all the features of your application and make sure everything is working before requesting a new review.